src/__stories__/display-media-card-story.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import {
+    DisplayMediaCard,
+    ButtonPrimary,
+    ButtonLink,
+    IconInvoicePlanFileRegular,
+    skinVars,
+    Circle,
+    Tag,
+    TagType,
+    ButtonSecondary,
+    IconLightningRegular,
+    ResponsiveLayout,
+    Stack,
+    Text2,
+    Inline,
+} from '..';
+
+export default {
+    title: 'Components/Cards/Display media card',
+};
+
+const BACKGROUND_SRC =
+    'https://images.unsplash.com/photo-1622819584099-e04ccb14e8a7?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1932&q=80';
```

Comment:
isn't it better to serve this new assets from storybook? to avoid the issue we had last week when this went down

Reply:
we have a task for that